packages/bot/src/scratch/blocks/Advanced/List/lists_repeat.js

```diff
@@ -22,19 +22,19 @@ Blockly.Blocks.lists_repeat = {
                     name: 'NUM',
                 },
             ],
-            colour           : Blockly.Colours.Binary.colour,
-            colourSecondary  : Blockly.Colours.Binary.colourSecondary,
-            colourTertiary   : Blockly.Colours.Binary.colourTertiary,
+            colour           : Blockly.Colours.Utility.colour,
+            colourSecondary  : Blockly.Colours.Utility.colourSecondary,
+            colourTertiary   : Blockly.Colours.Utility.colourTertiary,
             previousStatement: null,
             nextStatement    : null,
-            tooltip          : translate('List Repeat Tooltip'),
+            tooltip          : translate('Creates a list by repeating a given string or number'),
```

Comment:
This block can also take an array as input and repeat it, the tooltip contradicts this.

Reply:
fixed